American Whitewater/Info.plist

```diff
@@ -17,31 +15,22 @@
 	<key>CFBundlePackageType</key>
 	<string>APPL</string>
 	<key>CFBundleShortVersionString</key>
-	<string>1.1</string>
+	<string>$(MARKETING_VERSION)</string>
 	<key>CFBundleVersion</key>
-	<string>11</string>
-	<key>Fabric</key>
-	<dict>
-		<key>APIKey</key>
-		<string>b67e4513fcf72d81b5607976a8613ad0240f3105</string>
-		<key>Kits</key>
-		<array>
-			<dict>
-				<key>KitInfo</key>
-				<dict/>
-				<key>KitName</key>
-				<string>Crashlytics</string>
-			</dict>
-		</array>
-	</dict>
+	<string>$(CURRENT_PROJECT_VERSION)</string>
+	<key>ITSAppUsesNonExemptEncryption</key>
+	<false/>
 	<key>LSRequiresIPhoneOS</key>
 	<true/>
+	<key>NSLocationAlwaysUsageDescription</key>
+	<string>We use your location to show near-by rivers and other regional information.</string>
```

Comment:
Isn't it `nearby` rather than `near-by`? (Also applies below). If there's a question on this copy, run it by @AWKevin.

Reply:
fixed.